src/Entity/ShipmentServiceOptions.php

```diff
@@ -307,6 +350,33 @@ public function setNegotiatedRatesIndicator($NegotiatedRatesIndicator)
     /**
      * @return boolean
      */
+    public function isImportControlIndicator()
+    {
+        return isset($this->importControlIndicator);
+    }
+
+    /**
+     * @param boolean $importControlIndicator
+     * @return ShipmentServiceOptions
+     */
+    public function setImportControlIndicator($importControlIndicator)
+    {
+        $this->importControlIndicator = $importControlIndicator;
+        return $this;
+    }
+
+    /**
+     * @return ShipmentServiceOptions
+     */
+    public function unsImportControlIndicator()
```

Comment:
This method is not necessary. Using `setImportControlIndicator` with `null` will do the same.